### PR TITLE
feature reusable component

### PR DESCRIPTION
**Pull request by euledge**
We made multiple components at the same time.

The copyright was not explicitly stated, but can I use it for work?

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,3 +1,20 @@
 var Vue = new Vue({
-	el: 'body',
```
Comment (imrealashu) [inline script.js:2]: any specific reason for mounting the the root instance to `body` tag ? @euledge

**Review by imrealashu — CHANGES_REQUESTED** [review]
mounting the root instance to `body` or `html` tag is not recommended on vue 2. https://github.com/vuejs/vue/releases/tag/v2.0.0-rc.3

Please make mount it on a `div` tag as it was..

File: script.js (hunk 1 above)
Comment (euledge) [reply script.js:2]: I use mount to  `#app ` instead of  `body`.  
we know the way recommended on vue 2.
should split commit?

Comment (imrealashu) [reply script.js:2]: just change `el: 'body'` to `el: '#app'` in the source code.

File: script.js [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+	el: '#app',
```
Comment (euledge) [inline script.js:2]: looks like it has changed  `el: #app`

Comment (imrealashu) [reply script.js:2]: cool 👍

**Review by imrealashu — APPROVED** [review]

**Comment by imrealashu** [comment]
@euledge merged. also you can use it anywhere you want and PRs will be appreciated.

**Comment by euledge** [comment]
@imrealashu Thank you for merge. I 💛 Vue.js

**Comment by imrealashu** [comment]
are you on larachat ? let's have some word there ?

**Pull request merged**